BlueprintUICommonControls.podspec

```diff
@@ -14,4 +14,9 @@ Pod::Spec.new do |s|
   s.source_files = 'BlueprintUICommonControls/Sources/**/*.swift'
 
   s.dependency 'BlueprintUI'
+
+  s.test_spec 'Tests' do |test_spec|
```

Comment:
Added to get my tests running, need to figure out if we can actually add this back since it breaks pod linting.

Reply:
Still need to remove this.

Reply:
Removed